src/VisualStudio/IntegrationTests/CSharpInteractiveDemo.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class CSharpInteractiveDemo
+    {
+        private IntegrationHost _host;
+        private InteractiveWindow _interactiveWindow;
+
+        public CSharpInteractiveDemo(IntegrationHost host)
+        {
+            _host = host;
+
+            host.Initialize();
+
+            _interactiveWindow = host.CSharpInteractiveWindow;
+
+            _interactiveWindow.Show();
+            _interactiveWindow.Reset();
+
+            _interactiveWindow.SubmitTextToRepl(@"#r ""WindowsBase""
+#r ""PresentationCore""
+#r ""PresentationFramework""
+#r ""System.Xaml""", timeout: 20000);
+
+            _interactiveWindow.SubmitTextToRepl(@"using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;");
+        }
+
+        [Fact]
+        public void ScenarioOne()
```

Comment:
What guarantees that `ScenarioOne()` is executed before `ScenarioTwo()`?  Implementation order? If order doesn't matter then a descriptive name would be more helpful than a numeric one.